bessctl/bessctl

```diff
@@ -141,14 +142,15 @@ def main():
         line_buf = []
 
         for arg in sys.argv[1:]:
+            arg = arg.decode()
```

Comment:
py3k str is Unicode, yes. (The type name `unicode` no longer exists, it's all just `str`.) py2k str and bytes are the same underlying type. The py2k io.StringIO` does require unicode, so probably this should use `io.BytesIO` on py2k and `io.StringIO` on py3k. Or you can use `six.StringIO` here, if you're importing `six` (I forget if you are).